mysql-test/main/type_ranges.test

```diff
@@ -76,9 +76,11 @@ alter short drop default,
 DROP INDEX utiny,
 DROP INDEX ushort,
 DROP PRIMARY KEY,
-DROP FOREIGN KEY any_name,
```

Comment:
Can we add `IF EXISTS` here? I did not notice any test where we would do `DROP FOREIGN KEY IF EXISTS` on a non-existing name.